### ISSUE TITLE
Delete one-time keys where devices are deleted

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#3494]

**Comment by lampholder** [comment]
@ara4n Is this the device identity key? Or something else?

**Comment by ara4n** [comment]
no, the way Double Ratchet algorithms (e.g. Olm) work is that devices generate a pool of throw-away keypairs used when negotiating new encryption sessions.  They preload a bunch of them onto the server, so that folks wanting to start a new session will be able to pick one at random, after which it is discarded forever, making it harder to subvert the key exchange.

This bug is an e2e bug requesting that we vape stale one-time keys from old devices, as they just clog everything up and may cause perf probs.
